src/Indicator.jsx

```diff
@@ -13,24 +18,100 @@ class Indicator extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      activeArea: {},
+      selectedOption: null,
+      biomesByBlockData: [],
+      connError: false,
     };
   }
 
   componentDidMount() {
     const { activeArea } = this.props;
-    this.setState({ activeArea });
+    const areaId = (activeArea && activeArea.name) ? activeArea.name : null;
+    if (areaId) {
+      this.loadBiomes(areaId);
+    }
```

Comment:
I think you should add an else statement to set the connErrror state to false. But also this will be necessary only with a few indicators. This condition must change later.

Reply:
Done